easybuild/framework/easyblock.py

```diff
@@ -2231,7 +2242,7 @@ def cleanup_step(self):
         except when we're building in the installation directory or
         cleanup_builddir is False, otherwise we remove the installation
         """
-        if not self.build_in_installdir and build_option('cleanup_builddir'):
+        if not self.build_in_installdir and build_option('cleanup_builddir') and not build_option('continue'):
```

Comment:
well, if I properly interpret the intentions of my past self, the idea of `--continue` was in the context of a debug workflow, so it is assumed that one may need to continue to use `--continue` (pun intended)

once final, the changes should always be put in a patch and everything built from scratch.

but I have to say that my users have simply been debugging using `buildenv` or `--dump-env-script` on top of a manually prepared build dir, it's not exactly the same as this would allow but it fits most cases, so not sure if this is worth the trouble...